stl/inc/__msvc_all_public_headers.hpp

```diff
@@ -112,6 +112,7 @@
 #include <numeric>
 #include <optional>
 #include <ostream>
+#include <print>

```

Comment:
:warning: Note to self, this will require the usual internal changes when mirroring a new header.